apps/issuer/models.py

```diff
@@ -564,7 +564,37 @@ def get_user_or_none(recipient_id, recipient_type):
 
     return user
 
+class SuperBadge(BaseAuditedModelDeletedWithUser, BaseVersionedEntity):
+    entity_class_name = 'SuperBadge'
+    name = models.CharField(max_length=128)
+    description = models.CharField(max_length=255, blank=True)
 
+    assertions =   models.ManyToManyField('issuer.BadgeClass', blank=True,
+            through='issuer.SuperBadgeBadgeClass'
+        )  
+
+    # def delete(self, *args, **kwargs):
+    #     super(SuperBadge, self).delete(*args, **kwargs)
+
+    # def save(self, **kwargs):
+    #     super(SuperBadge, self).save(**kwargs)
+
+    @cachemodel.cached_method(auto_publish=True)
+    def cached_collects(self):
+        return self.assertions.all()
+   
+class SuperBadgeBadgeClass(cachemodel.CacheModel):
+    superbadge = models.ForeignKey('issuer.SuperBadge',
```

Comment:
I don't quite understand the relationship of the super badge to issuer / badge class, maybe a comment clarifying this would be helpful?